barchartrealtime.html

```diff
@@ -0,0 +1,41 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+ <title>Learning D3</title>
+  <style>
+  .h2{
+    text-align:left;
+    font-family:roboto;
+    font-size: 15px;
+  }
+  </style>
+  <h2 class='h3'>SMS Feedback Per County</h3>
+</head>
+<body>
+  
+    <div class="canvas">
+    </div>  
+
+<script src="https://www.gstatic.com/firebasejs/5.5.8/firebase-app.js"></script>
```

Comment:
The indentation is a bit jumbled here

Reply:
I _think_ the firebase imports should go into the header? I don't remember could you check?